java/src/jmri/jmrit/audio/swing/AudioBufferFrame.java

```diff
@@ -308,7 +299,8 @@ boolean applyPressed(ActionEvent e) {
     }
 
     private static int nextCounter() {
-        return counter++;
+        counter++;
+        return counter;
     }
```

Comment:
I'm not sure, but it seems to me that `counter` should be set to 0 instead of 1 on line 46 due to this change.